src/Compilers/Core/Portable/Operations/ConversionKind.cs

```diff
@@ -31,6 +31,11 @@ public enum ConversionKind
         /// </summary>
         OperatorMethod = 0x5,
         /// <summary>
+        /// Conversion is defined by the underlying type, and is created when an interpolated
+        /// string expression is being converted to an IFormattable or FormattableString.
+        /// </summary>
+        InterpolatedString = 0x6,
```

Comment:
I thought we decided in the design group meeting that since this conversion exists in the spec for both languages, we should make this one of the public types.